### ISSUE TITLE
gitpod extension (maybe a new one): allow integration with vscode.dev

### ISSUE THREAD

**Issue by GitMensch** [gitpod-io/gitpod#6618]
There are currently two gitpod extensions in the ms extension market, I think I fail to see from the README what they are actually for (the README seems to be about Gitpod, not about the extension).
That aside: It would be cool if there is an extension that allows using the existing gitpot workspaces in vscode.dev (providing Terminal/build/debug).

**Comment by akosyakov** [comment]
I don't think it is technically feasible. There is API between web workbench (vscode.dev) and VS Code Server running on the backend. It is not stable, i.e. it is only safe when web workbench and server are built from the same commit. Connecting arbitrary one likely is not going work and will break in subtle ways.

What could work if an extension which detects underlying git repo and suggest to open in Gitpod instead.

**Comment by GitMensch** [comment]
> What could work if an extension which detects underlying git repo and suggest to open in Gitpod instead.

If that would work also in a way that it creates the necessary workspace then this would be definitely useful.

Do you think you could clean up the extensions' README a bit to make more clear what the extensions actually do and if they are found at different repos adjust this content in the package.json (pointed all to the repo where I've created the issue).

**Comment by loujaybee** [comment]
I'm not sure I fully understand the use-case for having an extension that's compatible with vscode.dev, do you mind expanding a little on what you're trying to achieve with that @GitMensch ? 

Also, regarding the README's, you're right, it definitely is confusing, I've associated this issue with our GA of VSCode Desktop, we'll pick up this issue as part of that effort. Thanks a lot for raising that 🙏 .

Note: We might want to break this up into two issues.

**Comment by GitMensch** [comment]
> I'm not sure I fully understand the use-case for having an extension that's compatible with vscode.dev, do you mind expanding a little on what you're trying to achieve with that?

Sure. vscode.dev is now fully integrated in Github - open any path in the "code" tab and press a period "." opens vscode.dev at this place with the extensions of the user setup.
When I want to do actual work not just simple editing there's more needed - and it would be nice to have an extension I could install there that would open whatever I currently have open in a gitpod workspace.

**Comment by akosyakov** [comment]
I removed it for now from the groundwork, since we focus on initial onboarding in Dec.
